explanation/223-introduction.md

```diff
@@ -0,0 +1,18 @@
+(purpose-scope)=
+# Purpose and Scope
+
+The Purpose and Scope from from the current standard are provided below to introduce it to users. For more information about the standard, please see the published document [^1].
+
+[^1]: TODO link to public review.
+
+## 1. Purpose
+The purpose of this standard is to define formal knowledge concepts and a methodology to apply them to create interoperable, machine-readable semantic frameworks for representing building automation and control data, and other building system information.
+
+## 2. Scope
+This standard provides a comprehensive way to apply semantic formalisms to represent the context of building system data and relationships between the associated building mechanical system components so that software applications can find and understand the information in an automated way. It is intended to facilitate the development and implementation of building analytics tools and enterprise knowledge applications that can implement many building system functions, including:
+1. automated fault detection and diagnostics
+2. building system commissioning
+3. digital twins
+4. real time optimization
+5. smart grid interactions
```

Comment:
```suggestion
>This standard provides a comprehensive way to apply semantic formalisms to represent the context of building system data and relationships between the associated building mechanical system components so that software applications can find and understand the information in an automated way. It is intended to facilitate the development and implementation of building analytics tools and enterprise knowledge applications that can implement many building system functions, including:
>1. automated fault detection and diagnostics,
>2. building system commissioning,
>3. digital twins
>4. optimization of energy use, and
>5. smart grid interactions.
```